sdk/type.go

```diff
@@ -81,17 +87,15 @@ func (outputType *OutputType) GetScalingFactor() (float64, error) {
 	return strconv.ParseFloat(outputType.ScalingFactor, 64)
 }
 
-// Apply applies the transformations specified by the OutputType to
-// a reading value. These transformations are (in the order that they
-// are applied): multiply scaling factor.
-//
-// Precision is not applied at this level, but will instead be applied
-// in Synse server before the corresponding reading is returned to the
-// user.
-func (outputType *OutputType) Apply(value interface{}) interface{} { // nolint: gocyclo
+// applyScalingFactor multiplies the raw reading value (the value parameter) by the output
+// scaling factor and returnes the scaled reading.
+func (outputType *OutputType) applyScalingFactor(value interface{}) interface{} {
 	scalingFactor, err := outputType.GetScalingFactor()
 	if err != nil {
-		return value
+		log.Errorf(
+			"[type] Unable to get scaling factor for outputType %+v, error: %v",
+			outputType, err.Error())
+		return value // TODO: Return the error.
```

Comment:
Did not change the Apply prototype to return an error. This method is mostly factoring out the pre-existing apply.